mkudlaty-boston.md

```diff
@@ -0,0 +1,5 @@
+# Boston, MA
```

Comment:
I was in Boston a long time ago, when I was a teenager. I don't remember much, but I remember it is a cool city.